mysql-test/suite/galera/t/MDEV-28452.test

```diff
@@ -0,0 +1,98 @@
+#
+# Test for MDEV-28452: wsrep_ready: OFF after MDL BF-BF conflict
+#
+
+--source include/galera_cluster.inc
+--source include/have_debug.inc
+--source include/have_debug_sync.inc
+
+--connect node_2a, 127.0.0.1, root, , test, $NODE_MYPORT_2
+
+--connection node_1
+
+# create two tables
+CREATE TABLE `user` (
+  `id` char(36) COLLATE utf8mb4_unicode_ci NOT NULL,
```

Comment:
These column names are indeed copied exactly from the customer case. 
I have now removed all those columns that are not part of the keys and, thus, do not affect the test in any way.
But it would be possible to create a minimal test case with table names `t1` and `t2` and with just two columns per table.